src/connected/connections.ts

```diff
@@ -166,6 +166,15 @@ export class AllConnectionsTreeDataProvider implements VSCode.TreeDataProvider<C
     return null;
   }
 
+  async getParent(element: ConnectionsNode): Promise<ConnectionsNode> {
+    if (element.contextValue === 'sonarqubeConnection') {
+     return this.getInitialState()[0];
+    } else if (element.contextValue === 'sonarcloudConnection') {
+      return this.getInitialState()[1];
+    }
```

Comment:
I'm not sure how this works if there are only Server (resp. Cloud) connections.

Reply:
Ah yes, and in this case it should always be the non-null item (since it has a child) 👍🏻